src/routes/index.js

```diff
@@ -1,8 +1,28 @@
 import { Router } from 'express';
-import api from './api';
+import jwt from 'jsonwebtoken';
+import { findByEmail } from '../utils/searchDb';
+import routes from './api';
 
 const router = Router();
 
-router.use('/api/v1', api);
+router.use('/api/v1', routes);
+router.post('/api/v1/auth/login', async (req, res, next) => {
+  try {
+    const user = await findByEmail(req.body.email);
+    const { id, roleId } = user;
+    let token = await jwt.sign({ id, roleId }, process.env.JWT_SECRET);
+    token = `Bearer ${token}`;
+    res
+      .status(200)
+      .header('x-auth-access', token)
```

Comment:
Please use `authorization` for the token instead of `x-auth-access`

Reply:
Done